style.css

```diff
@@ -123,3 +123,188 @@ p {
   line-height: 32px;
   text-align: center;
 }
+
+.card {
+  height: 350px;
+  background-color: #ebf0ee;
+  flex-direction: column;
+  justify-items: center;
+  justify-content: space-around;
+  align-content: center;
+  border-radius: 6px;
+  display: grid;
+  margin-top: 20px;
+  margin-right: 25px;
+  margin-left: 25px;
+}
+
+h2 {
+  text-align: center;
+  color: #172b4d;
+}
```

Comment:
- [ ] Kindly implement the [required changes](https://github.com/JansueT/Portfolio/pull/2/files#r1038810448) from the previous reviewer. 

![Screenshot from 2022-12-04 01-24-54](https://user-images.githubusercontent.com/60083437/205458292-3a27b86a-e760-4033-9b25-0aa2b0898c9a.png)